validator/duty_execution.go

```diff
@@ -136,63 +137,66 @@ func (v *Validator) comeToConsensusOnInputValue(ctx context.Context, logger *zap
 		}
 		inputByts, err = json.Marshal(d)
 		if err != nil {
-			return 0, nil, nil, errors.Errorf("failed on attestation role: %s", role.String())
+			return 0, nil, 0, errors.Errorf("failed to marshal on attestation role: %s", role.String())
 		}
 		valCheckInstance = &valcheck.AttestationValueCheck{}
-	//case beacon.RoleAggregator:
-	//	aggData, err := v.beacon.GetAggregationData(ctx, slot, duty.GetCommitteeIndex())
-	//	if err != nil {
-	//		return 0, nil, errors.Wrap(err, "failed to get aggregation data")
-	//	}
-	//
-	//	inputValue.Data = &proto.InputValue_AggregationData{
-	//		AggregationData: aggData,
-	//	}
-	//case beacon.RoleProposer:
-	//	block, err := v.beacon.GetProposalData(ctx, slot)
-	//	if err != nil {
-	//		return 0, nil, errors.Wrap(err, "failed to get proposal block")
-	//	}
-	//
-	//	inputValue.Data = &proto.InputValue_BeaconBlock{
-	//		BeaconBlock: block,
-	//	}
-	default:
-		return 0, nil, nil, errors.Errorf("unknown role: %s", role.String())
-	}
+	case beacon.RoleAggregator:
```

Comment:
@nivBlox let's remove aggregator and proposal roles for now

Reply:
i sugget to leave it for the next pr to save us to write the same code
@lior-blox